src/smtencoding/FStar.SMTEncoding.Encode.fst

```diff
@@ -1000,6 +1000,547 @@ let encode_top_level_let :
       [decl] |> mk_decls_trivial, env
 
 
+let is_sig_inductive_injective_on_params (env:env_t) (se:sigelt)
```

Comment:
This function is hoisted out of the Sig_inductive case so that we can use its result for both the encoding of Sig_inductive and Sig_datacom